ftlib/__init__.py

```diff
@@ -77,10 +105,10 @@ def _rebuild(self):
         try:
             if self.framework.type == "dummy_NCCL":
                 if_success = self.framework.rebuild(self.rank, self.size)
-            if self.framework.type == "pytorch":
-                if_success = self.framework.rebuild(
-                    self.rank, self.size, master_addr=master_addr
-                )
+            if self.framework.type == 'pytorch':
```

Comment:
Do we have a standard for `'` and `"`? cc @terrytangyuan

Reply:
Yes. You can actually just run the pre-commit command or black command to fix this automatically.

Reply:
This command will fix many of the issues automatically: https://github.com/caicloud/ftlib/blob/master/.travis.yml#L23

Reply:
thanks guys, I've installed the pre-commit on my machine.